src/register/register.ts

```diff
@@ -146,10 +316,49 @@ export class Register {
         })
       );
 
-      Register.registers[register].text = text;
-    }
+      // Harmonize newline character
+      text = text.replace(/\r\n/g, '\n');
```

Comment:
Can we read `files.eol` to use the proper line ending of the file? https://code.visualstudio.com/Docs/customization/userandworkspace